docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -233,13 +233,20 @@ testcafe all tests/sample-fixture.js -r my-reporter
 
 The generated test report will be displayed in the command prompt window.
 
-If you need to save the report to an external file, you can redirect the command output stream to the file
-by using the > redirection operator and specifying the report file path.
+If you need to save the report to an external file, specify the file path after the report name.
 
 ```sh
-testcafe all tests/sample-fixture.js > tests/test-results.txt
+testcafe all tests/sample-fixture.js -r json:report.json
 ```
 
+You can also use multiple reporters in a single test run. List them separated by commas.
+
+```sh
+testcafe all tests/sample-fixture.js -r spec,xunit:report.xml
+```
+
+Note that only one reporter can write to `stdout`. All other reporters must output to files.
```

Comment:
Does the order of reporters matter? Can i use `xunit:report.xml,spec,nunit:report-nunit.xml` ?

Reply:
yes, you can. Do you think we should highlight it there?

Reply:
Yes, I mean we should add this info.

Reply:
I'm not sure that it's significant remark but as you wish /cc @VasilyStrelyaev

Reply:
I don't think we should add it. That's intuitive enough.
We don't write that we can specify browsers or pass command line parameters in any order. Why should we say that?

Reply:
Ok, then.